lib/ProductOpener/Ingredients.pm

```diff
@@ -5822,13 +5688,7 @@ sub replace_allergen_in_caps($$$$) {
 }
 
 
-sub replace_allergen_between_separators($$$$$$) {
-	my $language = shift;
-	my $product_ref = shift;
-	my $start_separator = shift;
-	my $allergen = shift;
-	my $end_separator = shift;
-	my $before = shift;
+sub replace_allergen_between_separators($language, $product_ref, $before, $start_separator, $end_separator, $allergen) {
```

Comment:
The arguments are not in the same order, so they will be mistmatched.

Reply:
Alright, sorry I didn't know that would/might cause errors